fitbenchmarking/utils/default_options.ini

```diff
@@ -92,11 +92,12 @@ use_errors: yes
 #              have the top ranking (brightest) and values over 3 will show as
 #              the worst ranking (deep red)
 # default thresholds are 1.1, 1.33, 1.75, 3, and inf
-colour_scale: 1.1, ranking-top-1
-              1.33, ranking-top-2
-              1.75, ranking-med-3
-              3, ranking-low-4
-              inf, ranking-low-5
+colour_scale: 1.1, ranking-top-1, #fef0d9
```

Comment:
Please mention that the colours are customisable in the above comment if they are options. Can you also add the current colours to the "default" line of the comments

Reply:
Done